backend/python/app/graphql/types/entity_type.py

```diff
@@ -0,0 +1,16 @@
+import graphene
+
+class EntityRequestDTO(graphene.InputObjectType):
```

Comment:
Is the difference in naming for the Entity DTOs / User DTOs intentional?

Reply:
I'm mainly following the patterns in `resources/`. As far as I can tell, UserDTO is the "response dto" for all user endpoints. Each user endpoint has its own "request dto" (i,e CreateUserDTO)